modules/tests/test-integration/pom.xml

```diff
@@ -48,44 +56,20 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>org.apache.ftpserver</groupId>
-            <artifactId>ftpserver-core</artifactId>
+            <groupId>org.wso2.orbit.org.yaml</groupId>
+            <artifactId>snakeyaml</artifactId>
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>org.wso2.orbit.org.yaml</groupId>
-            <artifactId>snakeyaml</artifactId>
+            <groupId>org.apache.mina</groupId>
+            <artifactId>mina-core</artifactId>
```

Comment:
I'm assuming this was earlier transitive dependency and you have added it explicitly?

Reply:
Yes